jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/unit/BadgeUnitTests.java

```diff
@@ -2,31 +2,31 @@
 
 import io.github.epam.TestsInit;
 import org.testng.annotations.BeforeMethod;
-import org.testng.annotations.Ignore;
 import org.testng.annotations.Test;
 
-import static io.github.com.StaticSite.angularPage;
-import static io.github.com.pages.sections.BadgeSection.textWithBadge;
-import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+import static com.jdiai.tools.Timer.waitCondition;
+import static io.github.com.StaticSite.badgePage;
+import static io.github.com.enums.Colors.DEEP_PURPLE;
+import static io.github.com.pages.BadgePage.textWithBadge;
 import static org.testng.Assert.assertTrue;
 
-// TODO Move to the new page
-@Ignore
 public class BadgeUnitTests extends TestsInit {
 
     @BeforeMethod(alwaysRun = true)
     public void before() {
-        shouldBeLoggedIn();
-        angularPage.shouldBeOpened();
+        badgePage.open();
+        waitCondition(() -> badgePage.isOpened());
+        badgePage.checkOpened();
+        textWithBadge.show();
     }
 
     @Test
     public void badgeTest() {
```

Comment:
This test are not angular specific, it is just JDI basic validations.

If you can not divide the test to unit test, then just remove unit tests at all